File: app/src/main/java/com/ole/life/mvp/splash/ui/activity/SplashActivity.java
package com.ole.life.mvp.splash.ui.activity;

import com.ole.life.R;
import com.ole.life.mvp.splash.contract.SplashContract;
import com.ole.life.mvp.splash.model.SplashModel;
import com.ole.life.mvp.splash.presenter.SplashPresenter;
import com.wlh.common.base.BaseMvpActivity;



/**
 * Created by on 10/15/2019 15:33
 * 版本: 1.0
 * 描述:
 * 修改说明:
 */
public class SplashActivity extends BaseMvpActivity<SplashModel, SplashContract.View, SplashPresenter> implements SplashContract.View {

    @Override
    public int onBindLayout() {
        return R.layout.activity_splash;
    }

    @Override
    public void initView() {

    }

    @Override
    public void initData() {

    }

    @Override
    public SplashPresenter initPresenter() {
        return new SplashPresenter(this);
    }

}

File: app/src/main/java/com/ole/life/mvp/splash/ui/activity/SplashActivity.kt
package com.ole.life.mvp.splash.ui.activity

import com.ole.common.base.BaseMvpActivity

import com.ole.life.mvp.splash.contract.SplashContract
import com.ole.life.mvp.splash.presenter.SplashPresenter
import com.ole.life.mvp.splash.model.SplashModel

import com.ole.life.R

/**
 * Created by on 09/18/2019 15:27
 * 版本: 1.0
 * 描述:
 * 修改说明:
 */
class SplashActivity : BaseMvpActivity<SplashModel, SplashContract.View, SplashPresenter>(), SplashContract.View {

    override fun onBindLayout(): Int {
        return R.layout.activity_splash
    }

    override fun initView() {

    }

    override fun initData() {

    }

    override fun initPresenter(): SplashPresenter {
        return SplashPresenter(this)
    }

}
